### PR TITLE
add gulpfile.js, bootstrap bower_components

### PR DESCRIPTION
**Pull request by Liadonne**

### CODE REVIEW

File: backend/index.js [hunk 1]
```diff
@@ -7,6 +7,6 @@ app.use(express.static(__dirname + '/../frontend'));
 app.listen(process.env.PORT || 5000, function(req, res) {
     console.log('Example app listening on port 5000!');
 });
-app.get('*', function(req, res) {
+app.get('*.html', function(req, res) {
```
Comment (kiyanov) [inline backend/index.js]: What was the reason of changing this?

Comment (Liadonne) [inline backend/index.js]: Server doesn't send css, server response the html

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    cssmin = require('gulp-cssmin'),
+    imagemin = require('gulp-imagemin'),
+    pngquant = require('imagemin-pngquant'),
+    rimraf = require('rimraf'),
+    browserSync = require("browser-sync"),
+    reload = browserSync.reload;
+
+
+    var path = {
+        build: { //Тут мы укажем куда складывать готовые после сборки файлы
```
Comment (kiyanov) [inline gulpfile.js]: Don't use Russian

File: gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+            js: 'frontend/build/js/',
+            css: 'frontend/build/css/',
+            img: 'frontend/build/img/',
+        },
+        src: { //Пути откуда брать исходники
```
Comment (kiyanov) [inline gulpfile.js]: Please use English language

File: backend/index.js (hunk 1 above)
Comment (kiyanov) [inline backend/index.js]: Leave it as it was for now, it should work correctly.

**Pull request merged**